README.md

```diff
@@ -2,3 +2,7 @@
 Awesome Tic Tac Toe
 
 made with Scala on my free time. Maybe one day this become big and nice =D
+
+#Autor 
+Lucas Cazarote and
```

Comment:
Please use lists instead of word and.
Example: 
https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#lists